model/riscv_softfloat_interface.sail

```diff
@@ -26,7 +26,6 @@
 /* All arguments and results have one of these types                */
 
 type bits_rm     = bits(3)    /* Rounding mode */
-type bits_fflags = bits(5)    /* Accrued exceptions: NV,DZ,OF,UF,NX */
```

Comment:
I can't see how this builds if you delete the type used by the unconverted functions...

Reply:
Oh, I see, you moved that but nothing else to a new file. That seems like a strange design choice; I would expect fflags and frm to be defined next to each other given they both constitute fcsr.

Reply:
This type moved to the `riscv_float_common.sail`, which will be leveraged by the floating-point implementation (like float_eq) in sail.

Given the softfloat_interface will be retired soon, it is better to let the `softfloat_interface` 
 depends on `riscv_float_common.sail`. Of course, you are right, the related types like frm/fflags will be removed to the `riscv_float_common.sail` soon. I didn't perform this change because they are not referenced by the float_eq.

Reply:
Updated, please help to review when you free.